src/main/java/org/opentripplanner/graph_builder/module/osm/MixinPropertiesBuilder.java

```diff
@@ -0,0 +1,51 @@
+package org.opentripplanner.graph_builder.module.osm;
+
+import org.opentripplanner.graph_builder.module.osm.specifier.OsmSpecifier;
+
+/**
+ * Builder for {@link MixinProperties}. If you don't set the safety features they will have a default
+ * value of 1, which means no change.
+ */
+public class MixinPropertiesBuilder {
+
+  private SafetyFeatures bicycleSafety = new SafetyFeatures(1d, 1d);
+  private SafetyFeatures walkSafety = new SafetyFeatures(1d, 1d);
+
+  public static MixinPropertiesBuilder ofWalkSafety(Number safety) {
```

Comment:
Why do we use Number instead of double here?

Reply:
I did this so you can pass in a float, double or int. However, after rechecking my code, I actually don't need it anymore. I've removed it.